packages/stargate/src/signingstargateclient.ts

```diff
@@ -72,6 +72,7 @@ import { calculateFee, GasPrice } from "./fee";
 import { DeliverTxResponse, StargateClient } from "./stargateclient";
 
 export const defaultRegistryTypes: ReadonlyArray<[string, GeneratedType]> = [
+  ["/cosmos.base.v1beta1.Coin", Coin],
```

Comment:
Here we also need `/cosmos.bank.v1beta1.MsgSend`. The `MsgMultiSend` we have below is an advanced version of that (multiple inputs and multiple outputs).